File: C#/Motorbike.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TollFeeCalculator
{
    public class Motorbike : Vehicle
    {
        public override bool IsTollFree()
        {
            return true;
        }
    }
}

File: PHP/src/Motorbike.php
<?php

namespace App;

/**
 * Defines a motorbike vehicle.
 */
class Motorbike implements VehicleInterface {

  /**
   * @inheritDoc
   */
  public function getType(): string {
    return 'Motorbike';
  }

  /**
   * @inheritDoc
   */
  public function isTollFree(): bool {
    return true;
  }

}
